e2e_tests/tests/fixtures/unmanaged/error_termination.py

```diff
@@ -3,15 +3,18 @@
 # Demonstrate an experiment which errors out.
 
 import random
+import tempfile
 
 from determined.experimental import core_v2
 
 
 def main():
+    checkpoint_storage = tempfile.mkdtemp()
```

Comment:
I don't see what this is for.  It seems like an odd thing to put in an example, what's it for?

Reply:
I demoted this file from being an example into being a fixture for a test, and I wanted to test the case where explicitly specify the checkpoint storage location.

Reply:
oh, I missed the demotion